BUILDING.md

```diff
@@ -2,11 +2,13 @@
 
 You need [git](https://git-scm.com/), [the Rust toolchain](https://www.rust-lang.org/tools/install), and your platform's standard C and C++ compilers.
 
+* On Windows, notice that you must run the Rust toolchain installer from a terminal instead of double clicking it.
```

Comment:
Are you sure it doesn't work otherwise? It's just an exe, it will open a command prompt surely?

Reply:
It does not, it just does nothing. When you open it from a terminal, it displays several options. So I think it just outputs it to `STDOUT` and closes straight ahead.

![image](https://user-images.githubusercontent.com/397851/217782236-7e2e5522-dd26-4820-8bf5-69f1b26832bc.png)

Reply:
Wow… I feel weird about documenting this here though, I consider building to be a more advanced use-case so I expect the user will already be familiar with the command-line in that case. It seems like there's already an issue for it in Cargo: https://github.com/rust-lang/rustup/issues/2598

Would you be okay with dropping this line? I know it's helpful, but there's probably a lot of other things we'd need to add if we wanted to help users install a full toolchain.

Reply:
Sure, no problem